src/Extension.ml

```diff
@@ -29,12 +29,19 @@ end
 module Instance = struct
   type t =
     { mutable client : LanguageClient.t option
+    ; statusBarItem : StatusBarItem.t
     ; duneFormatter : DuneFormatter.t
     }
 
-  let create () = { client = None; duneFormatter = DuneFormatter.create () }
+  let create () =
+    let statusBarItem =
+      Window.createStatusBarItem ~alignment:StatusBarAlignment.(tToJs Left) ()
+    in
+    statusBarItem ## command #= "ocaml.select-sandbox";
```

Comment:
We should avoid copying this string more than once in this module. Let's create a variable for it.